pixel-saver@deadalnix.me/convenience.js

```diff
@@ -0,0 +1,96 @@
+/* -*- mode:js; indent-tabs-mode:true; tab-width: 4 -*- */
```

Comment:
I assume this is coming as this. Please add a reference in README about where it come from and all.